src/main/kotlin/agallipo/BoardingCardSorter.kt

```diff
@@ -1,114 +1,49 @@
 package agallipo
 
-import java.io.ObjectInputFilter.merge
+class BoardingCardSorter : Journeys() {
```

Comment:
I think that extending here from Journeys is a bad usage of Inheritance, it is usually a common mistake. You inherit from a class when you want to have a more specific behavior. Here you are inheriting from Journeys to be able to use its methods in BoardingCardSorter.

Reply:
You're are right, I followed your advice and also change the name of the class as JourneysGenerator, do you think this name suits better?